bundle/tests/scorecard/kuttl/test-iqe-jobs-vault/02-delete.yaml

```diff
@@ -11,3 +11,29 @@ delete:
 - apiVersion: cloud.redhat.com/v1alpha1
   kind: ClowdEnvironment
   name: test-iqe-jobs-vault
+
+
+
+apiVersion: cloud.redhat.com/v1alpha1
+kind: ClowdJobInvocation
+metadata:
+ name: host-inventory-smoke
+ namespace: test-iqe-jobs
```

Comment:
I think perhaps this is in the wrong place and is the reason for the failure of the test, it tries to delete a CJI from a namespace that is for another test.

Reply:
Yup, this was an accidental edit